android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DDMStructure.java

```diff
@@ -34,15 +34,10 @@ public DDMStructure createFromParcel(Parcel in) {
 			return new DDMStructure[size];
 		}
 	};
+
 	protected List<Field> fields = new ArrayList<>();
 	protected Locale locale = Locale.US;
 	protected boolean parsed;
-	private String description;
-	private String name;
-	private String structureKey;
-	private String structureId;
-	private Long classNameId;
-	private String classPK;
 
```

Comment:
I would prefer creating getters for this rather than removing them entirely, what do you think?

Reply:
Those could be local variables in `parse` method (as I did with `classNameId` and `classPK`) but they are unused. So, I chose remove them. It's better to have getters?